modules/lib/src/main/resources/assets/js/app/view/context/widget/details/ScheduleWizardStepForm.ts

```diff
@@ -66,6 +84,20 @@ export class ScheduleWizardStepForm
         });
     }
 
+    private findFirstChildWithClassIn(parent: Element, className: string): Element {
```

Comment:
why does the class name end with "`In`"? "`findFirstChildWithClass`" should do just fine.

Reply:
Since we are passing the item we search in I added "In" postfix to diverse between inherited element functions that search within current element